packages/libs/client/src/pactCreator.ts

```diff
@@ -0,0 +1,52 @@
+import { createExp } from '@kadena/pactjs';
+
+import { parseType } from './utils/parseType';
+import { ContCommand } from './contPact';
+import { IPact, PactCommand } from './pact';
+
+import { log } from 'debug';
```

Comment:
this is not correct the [log function](https://github.com/debug-js/debug/blob/master/src/node.js#L193) is the pure stderr.write without any checking, you need to use the setup function instead.
```suggestion
import debug, { Debugger } from 'debug';

const log: Debugger = debug('pactjs:proxy');
```